pup-venv: Using venv instead of virtualenv sometimes does not include pip

We had the same issue with the mu virtualenv feature.

```
Packaging Mu into a macOS native application.
python -m venv venv-pup
# Don't activate venv-pup because:
# 1. Not really needed.
# 2. Previously active venv would be "gone" on venv-pup deactivation.
./venv-pup/bin/pip install pup
make: ./venv-pup/bin/pip: No such file or directory
make: *** [macos] Error 1
```

The venv does indeed not contain the pip exec:
<img width="886" alt="image" src="https://user-images.githubusercontent.com/4189262/107575759-2c045c00-6be8-11eb-84d1-b5bbd0067890.png">


Might be related to the creating this venv from a different venv. If I manually create a venv within my mu-venv I have the same  issue. But if I deactivate and try again it does add pip.

--- a/.gitignore
+++ b/.gitignore
@@ -82,3 +82,4 @@ target/
 .venv*/
 /mu/wheels/*.whl
 /baseline_packages.json
+venv-pup/

--- a/Makefile
+++ b/Makefile
@@ -109,7 +109,7 @@ macos: check
 	@echo "\nFetching wheels."
 	python -m mu.wheels
 	@echo "\nPackaging Mu into a macOS native application."
-	python -m venv venv-pup
+	python -m virtualenv venv-pup
 	# Don't activate venv-pup because:
 	# 1. Not really needed.
 	# 2. Previously active venv would be "gone" on venv-pup deactivation.

--- a/make.py
+++ b/make.py
@@ -321,7 +321,7 @@ def _build_windows_msi(bitness=64):
     if pup_pbs_url:
         os.environ["PUP_PBS_URL"] = pup_pbs_url
     cmd_sequence = (
-        [sys.executable, "-m", "venv", "venv-pup"],
+        [sys.executable, "-m", "virtualenv", "venv-pup"],
         ["./venv-pup/Scripts/pip.exe", "install", "pup"],
         [
             "./venv-pup/Scripts/pup.exe",

